tools/goctl/api/parser/g4/ast/importstack.go

```diff
@@ -0,0 +1,33 @@
+package ast
+
+import "errors"
+
+var ErrImportCycleNotAllowed = errors.New("import cycle not allowed")
```

Comment:
Exported vars and methods need to be commented to pass linter.

Reply:
Thanks, fixed it.